README.md

```diff
@@ -83,7 +83,7 @@ If you wanna use JuiceFS in Hadoop, check [Hadoop Java SDK](https://juicefs.com/
 
 - [Redis Best Practices](https://juicefs.com/docs/community/redis_best_practices)
 - [How to Setup Object Storage](https://juicefs.com/docs/community/how_to_setup_object_storage)
-- [Cache Management](https://juicefs.com/docs/community/cache_management)
```

Comment:
Will this break the link? If yes, we can update the title but keep the link

Reply:
handled by compatibility redirects: https://github.com/juicedata/docs/pull/520/files#diff-a038096cbdea434999e1dce5ab497212f1fe18204dde1a027ce3bdd663261a2aR187